### ISSUE TITLE
high contrast mode not enabled in windows high contrast for Firefox/Chrome

### ISSUE THREAD

**Issue by kwliou** [highcharts/highcharts#16873]
#### Expected behaviour
high contrast mode should be enabled in windows high contrast for Firefox/Chrome

#### Actual behaviour
high contrast mode is not enabled in windows high contrast for Firefox/Chrome

#### Live demo with steps to reproduce
Load accessibility module in high contrast mode

Hack to force it in other browsers:
https://jsfiddle.net/3uq2comg/

#### Product version

master

#### Affected browser(s)
Firefox, Chrome

**Comment by raf18seb** [comment]
Hi @kwliou, thank you for creating the ticket and PR!
Our accessibility ninja will review it soon.
